packages/psss-server/src/routes/reports/helpers.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { SYNDROMES } from '../../constants';
+
+export const validateSyndrome = (syndrome: unknown) => {
```

Comment:
```suggestion
export const validateSyndrome = (syndrome: unknown): asserts syndrome is string => {
```
Could even go further and start using string literal types for the syndromes... personally I think it could be really useful but would be a bit of an overhaul at the moment hehe

Reply:
We could do that, but to be honest if feels like a bit of an overkill, at least at this stage... Happy to consider it again in the future!